jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -492,10 +499,46 @@ class CodeWhispererService {
         psiFile: PsiFile,
         latencyContext: LatencyContext
     ): RequestContext {
-        val fileContextInfo = getFileContextInfo(editor, psiFile)
-        val caretPosition = getCaretPosition(editor)
+        // 1. file context
```

Comment:
nit: Do we need the bullet points comments?

Reply:
I would prefer keeping these for readability